eth_tester/backends/pyevm/main.py

```diff
@@ -339,8 +343,7 @@ def get_fork_block(self, fork_name):
             raise UnknownFork("Unknown fork name: {0}".format(fork_name))
 
     def configure_forks(self):
-        args, kwargs = _mk_fork_configuration_params(self.fork_config)
-        self.chain.configure_forks(*args, **kwargs)
+        pass
```

Comment:
Maybe this should raise an exception?  Silently passing seems weird.

Reply:
Thoughts on removing all the runtime fork configuration options? (`set_fork_block`, `get_supported_forks`, `get_fork_block`)

I'm not sure they work, and even if they do right now, I'm not sure it's a good idea to enable. That seems like the kind of thing that should be configured at setup time instead of runtime. That's the way py-evm would work most safely.

Reply:
So that comes with another quirk: Pyethereum 1.6 doesn't support Byzantium rules. I'm not keen to work hard to support a deprecated backend, so the current solution is to drop pyethereum1.6 support. Thoughts?

Reply:
Dropping all of pyethereum support is on my 1.0 tick-list so yes.